ccm_web/server/src/api/api.service.ts

```diff
@@ -91,4 +89,10 @@ export class APIService {
     const createSectionsApiHandler = new CreateSectionApiHandler(requestor, sections, course)
     return await createSectionsApiHandler.createSections()
   }
+
+  async enrollSectionUsers (user: User, sectionId: number, sectionUsers: SectionUserDto[]): Promise<CanvasEnrollment[] | APIErrorData> {
+    const requestor = await this.canvasService.createRequestorForUser(user, '/api/v1/')
+    const enrollmentHandler = new EnrollSectionUsersApiHandler(requestor, sectionUsers, sectionId)
+    return await enrollmentHandler.enrollUsers()
```

Comment:
Took me a bit to understand what @ssciolla was suggesting here, but I would agree that taking the users array as a parameter to `enrollUsers` makes sense to me design wise.  
I don't know that it will ever make a difference in how it's used, but it would allow the `EnrollSectionUsersApiHandler`  object to be reused, at least in theory, so it seems cleaner.

Reply:
I think I can support this idea, too.  I was following the pattern used for the create section code, which passed the input to the handler's constructor.  It didn't make sense to me why it was done that way, but seeing as it passed through code review, I thought that was the preferred technique.

Reply:
This is a good idea, but wasn't essential for completing this issue.  I'll open a new issue to implement this in the future.

Reply:
Opened #193.